### PR TITLE
Detect Dart files on Windows

### LINKED ISSUE #614: Dart files are not detected on Windows
Currently, Dart files are detected like this:

```dart
var dartFiles =
        await listFiles(pkgDir, endsWith: '.dart', deleteBadExtracted: true)
            .where((file) => file.startsWith('bin/') || file.startsWith('lib/'))
            .toList();
```

On Windows, this results in an empty list.
Because of this empty list, the dartanalyzer part isn't executed at all.

### PR DESCRIPTION
**Pull request by axel-op**
Fix #614.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: pana
 description: Evaluate the health and quality of a Dart package
-version: 0.13.5
```
Comment (isoos) [inline pubspec.yaml]: - please use 0.13.6-dev
- please run `pub run build_runner build` (to update the version.dart file)

**Comment by axel-op** [comment]
> please use 0.13.6-dev
> please run pub run build_runner build (to update the version.dart file)

@isoos Done 😊

File: pubspec.yaml [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+version: 0.13.6-dev
 homepage: https://github.com/dart-lang/pana
 
 environment:
-  sdk: '>=2.3.0 <3.0.0'
+  sdk: ">=2.3.0 <3.0.0"
```
Comment (axel-op) [inline pubspec.yaml]: I didn't notice before pushing, but my formatter replaced all these quotes... I hope this doesn't bother?

Comment (isoos) [reply pubspec.yaml]: It would be nice if you could revert those changes, thank you!

Comment (axel-op) [reply pubspec.yaml]: Also done

**Comment by axel-op** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed…

Sorry for that, I messed up by force-pushing the wrong commit. I rectified it.

**Comment by axel-op** [comment]
Everything should be OK by now.

Again, sorry for all of this...

**Comment by isoos** [comment]
@axel-op thanks!

**Pull request merged**